csharp-basics/exercises/FlowOfControl/FlowControl/LargestNumber/Program.cs

```diff
@@ -15,13 +14,20 @@ static void Main(string[] args)
 
             Console.WriteLine("Input the 3rd number: ");
             var input3 = Console.ReadLine();
-        
-            /*
-            todo - expected output:
-            Input the 1st number: 25
-            Input the 2nd number: 78
-            Input the 3rd number: 87
-             */
+         
+            if (Convert.ToInt32(input1) > Convert.ToInt32(input2) && Convert.ToInt32(input1) > Convert.ToInt32(input3))
```

Comment:
To konvertēšanu varētu veikt pirms šī un ielikt to kādā propertijā, ko pēc tam izmantot